src/sdk/namespace_fs.js

```diff
@@ -1111,6 +1112,27 @@ class NamespaceFS {
         }
     }
 
+    // handle read of specific part number from a multipart upload
+    // see _handle_complete_multipart_size()
+    _handle_read_part_number(params, res, stat) {
```

Comment:
Can you please add jsdoc on new functions? on _handle_complete_multipart_size() as well